### ISSUE TITLE
Bad header rive_canvas.js:2936 warning & Problem loading file; may be corrupt

### ISSUE THREAD

**Issue by Jackdaw16** [rive-app/rive-react#48]
I have a LoadingComponent with rive, the .riv file is placed in the root of the public folder.

![image](https://user-images.githubusercontent.com/13934225/145732378-31aed9eb-121f-422d-ad4b-9ce3cc56beb1.png)

This same component is called from two other components, the animation works correctly in the first component, but not in the second, the only clue I have are these two warnings.

![image](https://user-images.githubusercontent.com/13934225/145732495-ccc79760-6ebe-4466-aa68-3a1764b83696.png)

I'm not sure if I'm doing something wrong or if it's a bug from the library itself.

**Comment by avivian** [comment]
Hey @Jackdaw16, it's hard to tell exactly what the issue could be from your sample code, but I don't think it's that your .riv file is actually corrupt if it's working in one component. Not sure if this is a library bug or not but I suspect there could be some issue with the path in the `src` section of the params and the runtime is having issues loading the file. If that is the case we should have a better error message! 

How are you running/bundling your app? Would it be possible to create a small github repo that reproduces this issue with the two components? I've tried to reproduce the issue you've seen myself but can't seem to.

**Comment by Jackdaw16** [comment]
Hi, Here I attach some screenshots of how I have structured the application. The application is running with craco, for Tailwindcss. Between today or tomorrow I will do a simulation with a small project and create a Github repository.

what is indicated in red is where the library is used, the components and where the .riv is located.
![image](https://user-images.githubusercontent.com/13934225/145828328-0adc2521-85fa-4ef1-a710-bffd9d473d44.png)

**Comment by Jackdaw16** [comment]
I was try to reproduce the same error in a smaller project without getting the same result.
But, I tried such a simple thing, it seems silly, but it worked out.
I added two points to the route and it worked perfectly.

![image](https://user-images.githubusercontent.com/13934225/145922375-84f3caf3-dbc6-4396-b590-f1675854093b.png)

I don't really know why, because in the example I made with a single point, the animation worked just the same.
https://github.com/Jackdaw16/example-rive

but thank you very much and a really useful library, actually teach a little course on this library and Rive.
